src/functions/algolia-init-function.ts

```diff
@@ -11,21 +11,21 @@ import { Handler } from '@netlify/functions';
 import { DeliveryClient, IContentItem } from '@kontent-ai/delivery-sdk';
 import { canConvertToAlgoliaItem, convertToAlgoliaItem } from './utils/algoliaItem';
 import createAlgoliaClient from 'algoliasearch';
-import { hasStringProperty, nameOf } from './utils/typeguards';
+import { isValidInitRequestBody } from '../shared/types/initRequestBody';
 import { customUserAgent } from "../shared/algoliaUserAgent";
 import { createEnvVars } from './utils/createEnvVars';
 import { serializeUncaughtErrorsHandler } from './utils/serializeUncaughtErrorsHandler';
 import { sdkHeaders } from "./utils/sdkHeaders";
 
-const { envVars, missingEnvVars } = createEnvVars([ 'ALGOLIA_API_KEY' ] as const)
+const { envVars, missingEnvVars } = createEnvVars(['ALGOLIA_API_KEY'] as const)
 
 export const handler: Handler = serializeUncaughtErrorsHandler(async (event) => {
   if (event.httpMethod !== 'POST') {
     return { statusCode: 405, body: 'Method Not Allowed' };
   }
 
   const body = JSON.parse(event.body || 'null');
-  if (!isValidBody(body)) {
+  if (!isValidInitRequestBody(body)) {
     return { statusCode: 400, body: 'Missing or invalid body, please check the documentation' };
```

Comment:
Please extend to provide what is wrong the the body

Reply:
I extended the message to include all the property names found invalid (missing or of the wrong type).